src/components/FruitCounter.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from "./Button";
+
+function FruitCounter({ fruitType, buttonClicked, incrementCounter, decrementCounter }) {
```

Comment:
`buttonClicked` klinkt als een boolean waarde, maar het is een number. Ik zou het daarom een andere naam geven, zodat het duidelijker is welke waarde de variabele vast kan houden